tools/eslint-config-msal/.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://npm.pkg.github.com/AzureAD
```

Comment:
Tells npm to publish this package to Github packages, instead of the main npm registry.

Reply:
Does this not need the `@azuread` in front of it like in the main `./npmrc`?

Reply:
I'll can check if setting the scope will also publish it correctly.